test/factories/photos.rb

```diff
@@ -1,9 +0,0 @@
-FactoryGirl.define do
```

Comment:
I am putting all of the factories inside of the factories.rb file in the factories folder. I do not want us to have separate factory files inside of the factory folder. I'd rather us have just one factories.rb file.